src/features/pages/home/index.tsx

```diff
@@ -33,6 +34,7 @@ const HomePage = () => {
             <TradeTypes />
             <OurPlatforms />
             <WhatOurClientsSay />
+            <P2PBanner />
```

Comment:
```suggestion
      {is_p2p_allowed_country && <P2PBanner />}
```
please remove the old one from here as well, thanks :pray: